File: CS/DXSample/ViewModel/ViewModels.cs
using DevExpress.Mvvm;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DXSample.ViewModel {
    public class UserControlViewModel : ViewModelBase {
        private string _ViewName;
        public string ViewName {
            get {
                return _ViewName;
            }
            set {
                SetProperty(ref _ViewName, value, () => ViewName);
            }
        }
    }
}

File: VB/DXSample/ViewModel/ViewModels.vb
Imports DevExpress.Mvvm

Namespace DXSample.ViewModel

    Public Class UserControlViewModel
        Inherits ViewModelBase

        Private _ViewName As String

        Public Property ViewName As String
            Get
                Return _ViewName
            End Get

            Set(ByVal value As String)
                SetProperty(_ViewName, value, Function() ViewName)
            End Set
        End Property
    End Class
End Namespace
